### ISSUE TITLE
Error installing latest version with python 3.6

### ISSUE THREAD

**Issue by kekbait** [carpedm20/fbchat#207]
Looks like the recent changes made to requirements.txt (added conditional enum34 2.7 requirement) broke the installation process for me.

Command :
`sudo python3.6 -m pip install --upgrade fbchat`

Result :
```
Collecting fbchat
  Downloading fbchat-1.0.22.tar.gz
    Complete output from command python setup.py egg_info:
    error in fbchat setup command: 'install_requires' must be a string or list of strings containing valid project/version requirement specifiers; Invalid requirement, parse error at "'[:python'"
    
    ----------------------------------------
Command "python setup.py egg_info" failed with error code 1 in /tmp/pip-build-j17k8b_7/fbchat/

```

Thanks a lot for your amazing work !

**Comment by ekohilas** [comment]
This is currently being addressed in #204 
For the moment, you can download the previous version :)

**Comment by madsmtm** [comment]
I'm sorry, pip was autogenerating the file, should be fixed in `1.0.23`

**Comment by ekohilas** [comment]
Can confirm it has been fixed.

**Comment by madsmtm** [comment]
Assuming fixed since `1.0.23`, or at least by #213
